src/main/java/ch/njol/skript/classes/data/DefaultFunctions.java

```diff
@@ -404,6 +404,7 @@ else if (level >= 16 && level <= 30) // Truncating decimal parts probably works
 			}
 			
 		}.description("Calculates the total amount of experience needed to achieve given level from scratch in Minecraft.")
+				.examples("calcExperience(3) # Will return \"27\"", "send \"The max capacity of your level is %calcExperience(%level of player%)%\" to the player")

```

Comment:
```suggestion
				.examples("calcExperience(3) # Will return \"27\"", "send \"Your current level is %calcExperience(level of player)%\" to the player")
```

Reply:
should calculate including player's current exp

Reply:
oops yes